src/token/erc721/erc721.cairo

```diff
@@ -8,7 +8,8 @@
 #[starknet::component]
 mod ERC721Component {
     use openzeppelin::account;
-    use openzeppelin::introspection::dual_src5::{DualCaseSRC5, DualCaseSRC5Trait};
+    use openzeppelin::introspection::interface::ISRC5Dispatcher;
+    use openzeppelin::introspection::interface::ISRC5DispatcherTrait;
```

Comment:
We should merge this into one line import.

Reply:
definitely better